src/osd/ReplicatedBackend.cc

```diff
@@ -2223,19 +2223,19 @@ bool ReplicatedBackend::handle_push_reply(
 	error = true;
 	goto done;
       }
-      pi->recovery_progress = new_progress;
+      push_info->recovery_progress = new_progress;
       return true;
     } else {
       // done!
 done:
       if (!error)
-	get_parent()->on_peer_recover( peer, soid, pi->recovery_info);
+	get_parent()->on_peer_recover( peer, soid, push_info->recovery_info);
 
-      get_parent()->release_locks(pi->lock_manager);
-      object_stat_sum_t stat = pi->stat;
-      eversion_t v = pi->recovery_info.version;
+      get_parent()->release_locks(push_info->lock_manager);
+      object_stat_sum_t stat = push_info->stat;
+      eversion_t v = push_info->recovery_info.version;
       pushing[soid].erase(peer);
-      pi = NULL;
+      push_info = NULL;
```

Comment:
as this line is already touched... nullptr?

Reply:
:+1: